cli/tools/upgrade.rs

```diff
@@ -301,6 +313,7 @@ pub fn check_for_upgrades(
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct LspVersionUpgradeInfo {
   pub latest_version: String,
+  // TODO(bartlomieju): use `ReleaseChannel` instead
   pub is_canary: bool,
```

Comment:
Already asked @nayeemrmn to help me update this in a backwards compatible way in `vscode_deno`.

Reply:
I think this is fine as it is, `is_canary` just indicates whether or not `latest_version` refers to a commit hash (true) or an version tag (false). So I assume it could be true for `ReleaseChannel::Canary` and false otherwise.